### ISSUE TITLE
Customize auth details for Clickthrough pattern

### ISSUE THREAD

**Issue by ggeisler** [ProjectMirador/mirador#2638]
If the manifest auth `profile` value indicates the object should use the [`clickthrough` interaction pattern](https://iiif.io/api/auth/1.0/#clickthrough-interaction-pattern), there are some differences from the `login` pattern.

Use the following fallback strings when not values are not provided in the manifest:

- [ ] `label`: "Terms of use agreement required" (I'm not sure we need fallback text for this one, since `label` is "required" per the [auth spec](https://iiif.io/api/auth/1.0/#service-description), but perhaps we should add it just in case?).
- [ ] `header`: "You must agree to the terms of use to access this content."
- [ ] `description` : omit if not provided.
- [ ] `confirmLabel` : "Terms of use"
- [ ] `failureHeader`: "Terms of use not accepted"
- [ ] `failureDescription`: "You must agree to the terms of use to access this content."

Here are mockups for a scenario where none of the auth labels are provided in a clickthrough pattern:

<img width="1493" alt="terms-of-use-details" src="https://user-images.githubusercontent.com/101482/58724437-7486d200-8391-11e9-9189-1e02bbf4b1b3.png">

<img width="724" alt="terms-of-use-failure" src="https://user-images.githubusercontent.com/101482/58724445-7781c280-8391-11e9-99f8-d17ed404056c.png">

**Comment by cbeer** [comment]
I've intentionally left this out of #2640; I'm not sure we can/should assume that the clickthrough pattern is necessarily related to terms of service, and it feels a little funny to me to have fallback text that asks a user to accept some terms of service without being able to know what the image providers terms actually are.

**Comment by ggeisler** [comment]
Yeah, I see your point. I don't know what a clickthrough scenario other than one related to terms of service would be, unfortunately. That section of the spec assumes it is an agreement of terms scenario and offers no other examples. 

What do you think about something like this to make it more generic:

 `label`: "Action required" 
 `header`: "You must view additional information before accessing this content."
 `description`: omit if not provided.
 `confirmLabel`: "Continue"
 `failureHeader`: "Action required"
 `failureDescription`: "You must view additional information before accessing this content."

The failure strings just repeat the initial label and header.

**Comment by marlo-longley** [comment]
Audit 2026: for Auth 1 at least if a manifest doesn't supply these labels, the auth bar apparently renders with no text for those fields.
Should be evaluated with Auth 2.0 support work, not sure if clickthrough flow has changed.
Gary's more generic labels seem like reasonable fallbacks.

Depends on #3789
